ecosystem/typescript/sdk_v2/src/core/account.ts

```diff
@@ -20,9 +19,13 @@ import { Ed25519PrivateKey, Ed25519PublicKey, Ed25519Signature } from "../crypto
  */
 export class Account {
   /**
-   * A private key and public key, associated with the given account
+   * Public key associated with the account
    */
   readonly publicKey: PublicKey;
+
+  /**
+   * Private key associated with the account
+   */
   readonly privateKey: PrivateKey;
```

Comment:
I wonder if we should make this `optional`? As multi sig account does not have privateKey associated with it.

Reply:
That's next to determine